types/matter-js/index.d.ts

```diff
@@ -3702,18 +3751,32 @@ declare namespace Matter {
          * @event beforeUpdate
          * @param {} event An event object
          * @param {number} event.timestamp The engine.timing.timestamp of the event
+         * @param {number} event.delta The delta time in milliseconds value used in the update
```

Comment:
Incorrect, property is not available, see https://github.com/liabru/matter-js/blob/acb99b6f8784c809b940f1d2cf745427e088e088/src/core/Runner.js#L312-L317